### ISSUE TITLE
partnership nushell team?

### ISSUE THREAD

**Issue by unmellow** [sedwards2009/extraterm#358]
[nushell](https://www.nushell.sh/) is a new shell (pun intended by them) where commands can return tables
as opposed to raw text this seems like it would work well with your concept of frames maybe you could partner with
them? 

idk spit balling

**Comment by sedwards2009** [comment]
Yes, I know about nushell. It's really cool. Once I'm done with the migration from Electron to NodeGui/Qt I want to add direct support for showing the kind of tabular data that nushell deals in. i.e. like a mini-spreadsheet where you can sort/filter/hide colums/rows.
